batch_util_example.py

```diff
@@ -14,52 +14,69 @@
 from datetime import date, timedelta
 import demisto
 
+
 def format_dt(dt):
     return dt.strftime('%Y-%m-%dT%H:%M:%S')
 
+
 def options_handler():
-    parser = argparse.ArgumentParser(description='Utility for batch action on incidents')
-    parser.add_argument('-k', '--key', help='The API key to access the server', required=True)
-    parser.add_argument('-s', '--server', help='The server URL to connect to', required=True)
+    parser = argparse.ArgumentParser(
+        description='Utility for batch action on incidents')
+    parser.add_argument(
+        '-k', '--key', help='The API key to access the server', required=True)
+    parser.add_argument(
+        '-s', '--server', help='The server URL to connect to', required=True)
     monthAgo = date.today() - timedelta(days=30)
-    parser.add_argument('-f', '--filter', help='The filter query to chose the alerts, default is open incidents created in last month', default='(status:=0 or status:=1) and created:>%s' % format_dt(monthAgo))
-    parser.add_argument('-m', '--page', help='The page we are working on', default=0, type=int)
-    parser.add_argument('-n', '--size', help='The size per page', default=100, type=int)
-    parser.add_argument('-a', '--action', help='The action to perform', default='close', choices=['close'])
+    parser.add_argument('-f', '--filter', help='The filter query to chose the alerts, default is open incidents created in last month',
+                        default='(status:=0 or status:=1) and created:>%s' % format_dt(monthAgo))
+    parser.add_argument(
+        '-m', '--page', help='The page we are working on', default=0, type=int)
+    parser.add_argument(
+        '-n', '--size', help='The size per page', default=100, type=int)
+    parser.add_argument('-a', '--action', help='The action to perform',
+                        default='close', choices=['close'])
```

Comment:
If there is only one choice possible, what is the purpose of this argument? If default is set to `close` that means if someone doesn't use this option it is set to `close`. If someone does use this option, the only choice they can pick is `close`. If the only available option is `close` either way, what's the point of even having this option?

Reply:
create, export, close and delete are possible according to specification. in this example, only close is implemented. added explanation.(e37a8c8)